src/components/WhoWeAre/WhoWeAreDarkTheme.module.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../assets/styles/theme";
```

Comment:
nie rozbijaj na dwa pliki, zrób to w jednym i daj propsa np `isDarkTheme` true/false i od niego będzie zależeć jaka będzie klasa, znowu odsyłam do PR TheWoxPL

Reply:
@TheFenix2000 Ja bym powiedział, że rozwiązanie z dwoma plikami jest spoko, bo my potem jak będziemy wprowadzać, zmianę theme-ów globalną, to łatwo i szybko sobie to dostosujemy do sposobu, który wybierzemy. Moim zdaniem nie ma sensu, aby wprowadzać coś co może się nie przydać jak np. props isDarkTheme, i tak raczej użyjemy kontekstu lub zrobimy to w jakiś inny sposób. Co myślisz Michał?

Reply:
na razie to nieustalone z tym kontekstem jak będziemy robić w komponentach, wiec na razie niech będzie to zrobione na propsie tak jak reszta co nie, a potem jak juz będziemy mieli jasną wersję to sie zastosuje wszędzie, co do dwóch plików raczej reszta co była teraz sprawdzana robiła w jednym wiec niech tak zostanie też